recipe/meta.yaml

```diff
@@ -0,0 +1,177 @@
+{% set kx_license_available = False %}
+# this switch should be enabled if kdb can be downloaded as a test dependency
+{% set install_kdb_as_test_dep = False %}
+# # this switch enables experimental win-64 build depending on vs2010_runtime
+{% set winbuild_enabled = False %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/KxSystems/pykx
+  git_rev: release/{{ version }}
+  patches:
+    - 0001-Addition-of-missing-file.patch
+    - 0002-remove-linker-flags.patch
+
+build:
+  # skipping s390x and ppc64le because kx does not support these with their pre-built binaries
+  # skipping win because upstream q.dll and e.dll depend on an older MSVC runtime Anaconda no longer supports
+  skip: True  # [s390x or ppc64le]
+{% if not winbuild_enabled %}
+  skip: True  # [win]
+{% endif %}
+{% if not kx_license_available %}
+  script_env:
+    - QARGS="--unlicensed"
+{% endif %}
+  script:
+    # avoid distributing unnecessary shared objects depending on the platform
+    - if [ -d $SRC_DIR/src/{{ name }}/lib/m64arm ]; then rm -rv $SRC_DIR/src/{{ name }}/lib/m64arm; fi  # [(osx and x86_64) or linux]
+    - if [ -d $SRC_DIR/src/{{ name }}/lib/m64 ]; then rm -rv $SRC_DIR/src/{{ name }}/lib/m64; fi        # [(osx and arm64) or linux]
+    - if [ -d $SRC_DIR/src/{{ name }}/lib/w64 ]; then rm -rv $SRC_DIR/src/{{ name }}/lib/w64; fi        # [linux or osx]
+    - if [ -d $SRC_DIR/src/{{ name }}/lib/l64arm ]; then rm -rv $SRC_DIR/src/{{ name }}/lib/l64arm; fi  # [(linux and x86_64) or osx]
+    - if [ -d $SRC_DIR/src/{{ name }}/lib/l64 ]; then rm -rv $SRC_DIR/src/{{ name }}/lib/l64; fi        # [(linux and aarch64) or osx]
+{% if winbuild_enabled %}
+    - del /q /s %SRC_DIR%\src\{{ name }}\lib\l64 %SRC_DIR%\src\{{ name }}\lib\l64arm %SRC_DIR%\src\{{ name }}\lib\m64 %SRC_DIR%\src\{{ name }}\lib\m64arm  # [win]
+{% endif %}
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+    # fixing needed SO path on pre-built libobjstor delivered from upstream for linux
+    - patchelf --replace-needed '$ORIGIN/libkurl.so' python$PY_VER/site-packages/{{ name }}/lib/l64/libkurl.so $SP_DIR/{{ name }}/lib/l64/libobjstor.so # [linux and x86_64]
+{% if winbuild_enabled %}
+    # set Path variable to resolve location of q.dll
+    - set PATH=%SP_DIR%\{{ name }}\lib\w64;%PATH%  # [win]
+  missing_dso_whitelist:   # [win]
+  # both present in this package or vs2010_runtime but conda-build doens't find them
+    - $RPATH/MSVCR100.dll  # [win]
+    - $RPATH/q.dll         # [win]
+{% endif %}
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - git       # [not win]
+    - patch     # [not win]
+{% if winbuild_enabled %}
+    - m2-patch  # [win]
+{% endif %}
+    - patchelf  # [linux and x86_64]
+  host:
+    - python
+    - cython 3.0.0
+    - numpy 1.22  # [py!=37]
+    - numpy 1.20  # [py==37]
```

Comment:
```suggestion
    - numpy {{ numpy }}
```

Reply:
will do.

Reply:
actually, it seems that it really needs `numpy 1.22`, `conda_build_config.yaml `uses either `1.21` or `1.23`.

Reply:
What leads you to that conclusion (that it really needs 1.22)?